app/renderer/components/task_card.js

```diff
@@ -77,8 +78,21 @@ module.exports = {
 
         },
         deleteTask() {
-            this.stop();
-            this.$emit('remove');
+            SweetAlert.toggleModal(
```

Comment:
The class Alert (or SweetAlert) already scopes the action to a modal (or alert) so I think the method `toggleModal` should be rewritten simply as `toggle`